packages/medusa/src/services/__tests__/sales-channel.ts

```diff
@@ -162,35 +163,31 @@ describe("SalesChannelService", () => {
       manager: MockManager,
       eventBusService: EventBusServiceMock as unknown as EventBusService,
       salesChannelRepository: salesChannelRepositoryMock,
-      storeService: StoreServiceMock as unknown as StoreService
+      storeService: StoreServiceMock as unknown as StoreService,
     })
 
     beforeEach(() => {
       jest.clearAllMocks()
     })
 
-    it('should soft remove a sales channel', async () => {
+    it("should soft remove a sales channel", async () => {
```

Comment:
suggestion: we should probably add a new unit test on deleting the default sales channel